.prettierrc

```diff
@@ -1,7 +1,4 @@
 {
```

Comment:
Why change these? I do like how they are now, particularly the 4 spaces

Reply:
I prefer 2 because 4 makes lines very long...
Let vote! 
👍  for 4
👎  for 2

Reply:
I also prefer 2, with 4 spaces even 2-3 levels of nesting makes pretty much every line too long, especially with descriptive param/function names

Reply:
I see no reason to change it (so, implicitly it means 4).
Either way, this change can be in a different PR as it's just a matter of style and not code quality.

Reply:
In general, I'm pretty natural around this and don't see it as important, so if it's matter to you that much, change it to 2 (again, preferably in a different PR)

Reply:
Whatever 🤷 not gonna argue about this